components/nimbus/src/nimbus.udl

```diff
@@ -13,6 +13,7 @@ dictionary AppContext {
     string? os_version;
     string? android_sdk_version;
     string? debug_tag;
+    record<DOMString, string>? specific_context;
```

Comment:
`DOMString` here is a awkward, but I think that's a uniffi limitation? I'll double-check, saw it in the docs

Reply:
Not too sure about the name `specific_context`, happy to take suggestions for something more representative

Reply:
I don't have any issues with `specific_context`, maybe adding `app_` to it might make it more clear?  So `app_specific_context` or even `app_defined_context`?  Again, not blocking on this (naming things is hard).